src/core/api/project.go

```diff
@@ -403,12 +404,14 @@ func (p *ProjectAPI) List() {
 				return
 			}
 			projects = append(projects, pros...)
-			mps, err := p.SecurityCtx.GetMyProjects()
-			if err != nil {
-				p.SendInternalServerError(fmt.Errorf("failed to list projects: %v", err))
-				return
+			if sc, ok := p.SecurityCtx.(*local.SecurityContext); ok {
```

Comment:
Is this intentional?  Previously it doesn't require the securityContext must be local based.